x-pack/plugins/apm/typings/es_schemas/APMDoc.ts

```diff
@@ -4,11 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+export type AgentName =
```

Comment:
Can you add a comment here explaining that we _know_ this could be other values in theory, etc (like we talked about) :D

Reply:
Done